CMakeLists.txt

```diff
@@ -103,66 +93,66 @@ if (WITH_OPENSSL)
 endif ()
 
 
-function(add_hazelcast_library name type)
+function(add_hazelcast_library type)
```

Comment:
I guess now that we work with a single type of library during the build, we can remove this function and put its contents in the global scope.

Reply:
Removed.